CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Detekt - Changelog
 
+#### 2.0.0
+
+- Rewrite to sonar-detekt dropping redundant features to official SonarKotlin plugin
```

Comment:
This sentence sounds weird. Can we rewrite it to have two separate sentences in order to understand the change log in a better more meaningful way?